pyproject.toml

```diff
@@ -33,24 +35,30 @@ classifiers = [
     "Topic :: Scientific/Engineering",
 ]
 
+
 [tool.poetry.urls]
-"Tracker" = "https://github.com/N-Wouda/pyvrp/issues"
+"Tracker" = "https://github.com/N-Wouda/PyVRP/issues"
+
 
 [tool.poetry.dependencies]
 python = "^3.8,<4.0"
 numpy = ">=1.15.2"
 matplotlib = ">=2.2.0"

```

Comment:
In general related to dependencies, given that dependencies for benchmark, dev and examples are separated, does it make sense to have matplotlib a required dependency or should this be part of examples (for notebooks) and dev (for tests) groups?

Reply:
Since we have code that directly depends on matplotlib being installed, we should probably consider it a core dependency (e.g., when a user wants to use `pyvrp.plotting` they'll need matplotlib).

Reply:
Since we provide `pyvrp` as a CLI command, we should perhaps consider the benchmark group as core dependencies as well.

Reply:
Agree.